backend/pretix/__init__.py

```diff
@@ -383,4 +384,38 @@ def get_user_tickets(conference: Conference, email: str):
         },
     )
 
+    response.raise_for_status()
+
+    return response.json()
+
+
+def get_user_ticket(conference: Conference, email: str, id: str):
+
+    # TODO: filter by orderposition in the PretixAPI
+    tickets = get_user_tickets(conference, email)
+
+    def filter_by(ticket):
+        return str(ticket["id"]) == id and ticket["attendee_email"] == email
+
+    tickets = list(filter(filter_by, tickets))
+
+    if tickets:
+        return tickets[0]
+
+
+def is_ticket_owner(conference: Conference, email: str, id: str) -> bool:
+    ticket = get_user_ticket(conference, email, id)
```

Comment:
we should add a new API in pretix that checks the ownership

this API fetches lots more information than we need

but this is for after we merge this

Reply:
agree